firmware/brake/kia_soul_ev/tests/features/step_definitions/checking_faults.cpp

```diff
@@ -1,15 +1,3 @@
-WHEN("^a sensor becomes temporarily disconnected$")
```

Comment:
![oh god](https://orig00.deviantart.net/3f79/f/2014/012/e/6/oh_god_no_by_sonicfan1143-d71xvao.jpg)

Reply:
While I thought that this comment would be hilarious and innocuous, it looks like this was a workaround for a real bug. Testing this change on the Soul petrol demonstrated periodic sensor faults due to the sensor reads returning zero. Further investigation revealed noise induced on the sensor lines which was causing the voltage to drop below zero, causing the "bad" reads. While the changes in this PR are good it does exacerbate the issues that the behavior this code tested was papering over.

I'll get oscilloscope screenshots tomorrow; right now I can't locate any thumb drives and it's getting pretty late.

@rebpdx @snewt we should talk

Reply:
A second PR will be added for adding hysteresis as a fix.